admin/app/controllers/workarea/admin/bulk_variant_saving.rb

```diff
@@ -23,6 +23,7 @@ def save_variant_on_product(product, attributes: {}, variant: nil)
         if attributes[:inventory].present?
           inventory = Inventory::Sku.find_or_create_by(id: variant.sku)
           inventory.available = attributes[:inventory]
+          inventory.policy = 'standard'
```

Comment:
So it looks like this is used in sequential editing as well, where the `Inventory::Sku` might already exist and so this would override whatever it was already set to. Can you put a check on this to only do this if the inventory is new?

Side note, is there a reason it uses `find_or_create_by` instead of `find_or_initialize_by`? That's just doing an extra db write.

Reply:
No idea, but I was thinking the same thing. Mind if I change that to `.find_or_initialize_by`?